### DISCUSSION TITLE
Add aggregated view of linked collection objects

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5840] [Feature Requests]
> <p data-pm-slice="0 0 []">We have created an aggregation view for linked right-side collectionobject, which is working fine:</p>
> 
> ![image](https://github.com/specify/specify7/assets/37256050/f70c62f3-233f-4a7e-9035-aa2ef7593918)
> 
> 
> <p><br></p><p>As we have multiple linked collection we added the relationship name to it, which display like this:</p>
> 
> ![image](https://github.com/specify/specify7/assets/37256050/f3d0a1c0-2f35-47de-96d9-bbfdd5cea7a6)
> 
> 
> <p><br></p><p>However, we are wondering if there is a way to group the objects belonging to a relationship name (and possibly even an option to shorten the name) to improve visibility of the data. So instead of the above, we would have:</p>
> 
> Catalog Number | Left Side Rels
> -- | --
> H 6445-01 | Tissue: GT 1086 \| GT 14383 - Xrays: 5112 \| 5108 \| 5107 \| 5106 \| 5113
> 
> <p>&nbsp;<strong>Aggregation definition</strong></p>
> 
> ![image](https://github.com/specify/specify7/assets/37256050/b1129ac7-7bfb-4ed6-8153-87b33aba8077)

**Requested By:** Corinna P at CSIRO ([Asana](https://app.asana.com/0/1201972579963460/1204664224921509/f))

**Comment by maxpatiiuk** [comment]
This is possible via a workarounds.

Keep in mind that in 7.9 we are aiming to provide a visual editor for configuring aggregators and formatters which would make configuring these things simpler.

A workaround is to create separate aggregators - one for each prep type.
Each aggregator calls a separate formatter
The corresponding formater has a `switch`, making formatter only output content if it is of correct prep type
